eng/pipelines/mgmt-auto-release.yml

```diff
@@ -0,0 +1,54 @@
+trigger: none
+
+pr: none
+
+schedules:
+  - cron: "0 1 * * *"
+    displayName: Daily release
+    branches:
+      include:
+        - main
+    always: true
+
+stages:
+  - stage:
+    jobs:
+      - job:
+        displayName: Go Auto Release
+        steps:
+          - task: Npm@1
+            inputs:
+              command: 'custom'
+              customCommand: 'install -g autorest@latest'
+
+          - task: GoTool@0
+            inputs:
+              version: '1.18.2'
+
+          - script: |
```

Comment:
Ideally this would get broken out into its own script so changes in functionality can be more easily tested.

Reply:
Moved scripts section to /eng/scripts/mgmt-auto-release.sh